src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -41,6 +45,21 @@ public partial class UpdateDurableFunction
         logger = context.CreateReplaySafeLogger(EnsureArg.IsNotNull(logger, nameof(logger)));
 
         UpdateCheckpoint input = context.GetInput<UpdateCheckpoint>();
+        Uri uri = new Uri(input.Endpoint, UriKind.Absolute);
+        var callerClaims = new List<KeyValuePair<string, string>>
+        {
+            new ("operation_id", context.GetOperationId().ToString()),
+        };
+        _auditLogger.LogAudit(
+            AuditAction.Executing,
+            AuditEventSubType.UpdateStudy,
+            uri,
+            null,
+            Activity.Current?.RootId,
+            null,
+            callerClaims,
+            null);
+        HttpStatusCode auditStatusCode = HttpStatusCode.BadRequest;
```

Comment:
Both `HttpStatusCode` and `Url` are in this log? Is this an appropriate log for this kind of operation?